client/galaxy/scripts/mvc/ui/ui-misc.js

```diff
@@ -78,6 +78,57 @@ define(['utils/utils',
         }
     });
 
+
+    	var Password = Backbone.View.extend({
```

Comment:
Is this a copy of the above regular input element? I think we can reuse it, instead of copying it and just change the parsed type property from "text" to "password".